pkg/generated/crds/manifests/velero.io_backups.yaml

```diff
@@ -3,6 +3,8 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (unknown)
```

Comment:
Note this will be unknown because we use `go run` and aren't using Go modules to build it. See https://github.com/kubernetes-sigs/controller-tools/pull/338/files#diff-bcee38a2cd2e5f1990f046729f3833d8L22. I'm not sure if there's a way to have `go run` set the correct version.

Reply:
hmm, ok. seems harmless for now -- xref #1886